mage_ai/cluster_manager/config.py

```diff
@@ -65,4 +84,16 @@ def parse_config(self, config: Dict = None) -> Dict:
                 config=termination_policy
             )
 
+        post_start = config.get('post_start')
```

Comment:
You don't need to manually parse the subclass. This is already done in the base class `load` method: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/shared/config.py#L41-L46